tests/bsim/bluetooth/mesh/src/mesh_test.c

```diff
@@ -162,6 +164,15 @@ static struct bt_mesh_health_srv health_srv;
 static struct bt_mesh_model_pub health_pub = {
 	.msg = NET_BUF_SIMPLE(BT_MESH_TX_SDU_MAX),
 };
+static const uint8_t health_tests[] = {
```

Comment:
Hm. Is it used anywhere?

Reply:
Few lines below, to populate health metadata.

Reply:
It is needed in the LCD test. We do have tests for LCD models where they query various pages.